master/internal/db/postgres_filters.go

```diff
@@ -56,6 +56,16 @@ func filterToSQL(
 		_, _ = fragment.WriteString(strings.Join(paramFragments, ","))
 		_, _ = fragment.WriteString(")")
 		return fmt.Sprintf(fragment.String(), field)
+	case api.FilterOperationInOrNull:
+		var fragment strings.Builder
+		_, _ = fragment.WriteString("AND %s IS NULL OR %s IN (")
+		var paramFragments []string
+		for i := range values {
+			paramFragments = append(paramFragments, fmt.Sprintf("$%d", paramID+i))
+		}
+		_, _ = fragment.WriteString(strings.Join(paramFragments, ","))
+		_, _ = fragment.WriteString(")")
+		return fmt.Sprintf(fragment.String(), field)
```

Comment:
is this a bug, or am i reading it wrong?
```suggestion
		return fmt.Sprintf(fragment.String(), field, field)
```

Reply:
100%, forgot to update this when re-copying from `FilterOperationIn` .  Fixed.